include/onnxruntime/core/common/logging/logging.h

```diff
@@ -18,8 +18,11 @@
 #include "core/common/logging/severity.h"
 #include "core/common/logging/sink_types.h"
 #include "core/platform/ort_mutex.h"
-#include "date/date.h"
 
+// date.h is from abseil-cpp, which is no longer supporing C++20. So we need replace it with std::chrono::operator<<
```

Comment:
date.h is not from abseil-cpp. It is from https://github.com/HowardHinnant/date.